kornia/jit/__init__.py

```diff
@@ -4,3 +4,7 @@
 # expose functions to torch.jit
 # TODO: find an automatic way to do this
 rgb_to_grayscale = torch.jit.script(K.color.rgb_to_grayscale)
+
+spatial_soft_argmax2d = torch.jit.script(K.geometry.spatial_soft_argmax2d)
+spatial_softmax_2d = torch.jit.script(K.geometry.dsnt.spatial_softmax_2d)
+spatial_expectation_2d = torch.jit.script(K.geometry.dsnt.spatial_expectation_2d)
```

Comment:
I would test this API alongside tracing

Reply:
Aren't I doing this already? Each of these have test cases.

Reply:
if this it's true then it's fine. Still figuring out who jit should be properly tested. To me it's more intuitive through scripting API but really up to you :)